fuel-kotlinx-serialization/src/main/kotlin/com/github/kittinunf/fuel/serialization/FuelSerialization.kt

```diff
@@ -9,28 +9,33 @@ import com.github.kittinunf.fuel.core.response
 import com.github.kittinunf.result.Result
 import kotlinx.io.InputStream
 import kotlinx.io.Reader
-import kotlinx.serialization.KSerialLoader
+import kotlinx.serialization.DeserializationStrategy
+import kotlinx.serialization.ImplicitReflectionSerializer
 import kotlinx.serialization.json.JSON
 import kotlinx.serialization.serializer
 
+@ImplicitReflectionSerializer
 inline fun <reified T : Any> Request.responseObject(
-    loader: KSerialLoader<T> = T::class.serializer(),
+    loader: DeserializationStrategy<T> = T::class.serializer(),
```

Comment:
This might conflict with #485 though.

Reply:
I made the same change I think. Should merge fine.